packages/uma-auth-client/src/hooks/useOAuth.tsx

```diff
@@ -0,0 +1,100 @@
+import {auth, nwc, webln} from '@getalby/sdk';
+import { useEffect, useState } from "react";
+import { getUmaDomain } from "src/utils/getUmaDomain";
+import { useUser } from "./useUser";
+import { create } from 'zustand';
+import { persist } from "zustand/middleware";
+import { getNwcUmaDomain } from 'src/utils/getNwcUmaDomain';
+
+type UmaAuthToken = {
+  access_token: string,
+  refresh_token: string,
+  expires_at: number,
```

Comment:
It looks like this field is actually `expires_in` which I think is conventional for oauth.

Reply:
this is the type definition in alby:
```
export interface GetTokenResponse {
    /** Allows an application to obtain a new access token without prompting the user via the refresh token flow. */
    refresh_token?: string;
    /** Access tokens are the token that applications use to make API requests on behalf of a user.  */
    access_token?: string;
    token_type?: string;
    expires_in?: number;
    /** Comma-separated list of scopes for the token  */
    scope?: string;
}
export interface Token extends Omit<GetTokenResponse, "expires_in"> {
    /** Date that the access_token will expire at.  */
    expires_at?: number;
}
```
So it seems like it should be expires_at unless i'm misunderstanding something